index.yaml

```diff
@@ -12,9 +12,9 @@ entries:
       catalog.cattle.io/release-name: fleet
     apiVersion: v2
     appVersion: 0.3.5
-    created: "2021-06-11T16:24:22.11119-07:00"
+    created: "2021-06-21T15:12:36.065586-07:00"
     description: Fleet Manager - GitOps at Scale
-    digest: 2c1aadc3bf10e1d5d732ebc6a8a52730f33c9d159362561df9ac2f5d5b0a493c
+    digest: 6d0dbc8d82b1a485136dbb3588bf58bc88b610ecc7901557c6a65d70f12ec87b
```

Comment:
@jiaqiluo and I have seen these digests changing on regenerating the index.yaml for certain charts but have not been able to isolate why this is the case. This is a known issue but should not impact this PR.